metadata/contributors.json

```diff
@@ -4502,6 +4502,16 @@
          "keishi"
       ]
    },
+   {
```

Comment:
I think - if you can land this separately. It would be good. It will also give you option to use 'buildbot' (rebuild' button to restart failing bots (to check if there is any flaky test failures) etc.